<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlTabWidget</class>
 <widget class="QWidget" name="ControlTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetControl">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Arial&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="FrameWidget" name="tabFrame">
      <attribute name="title">
       <string>监控预览</string>
      </attribute>
     </widget>
     <widget class="TableWidget" name="tabTableWidget">
      <attribute name="title">
       <string>设备列表</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\framecontrol\framewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\tablecontrol\tablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
